geth/log/custom/custom.go

```diff
@@ -0,0 +1,172 @@
+package custom
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/fatih/color"
+	"github.com/status-im/status-go/geth/log"
+)
+
+// FlatDisplay writes giving Entries as seperated blocks of contents where the each content is
+// converted within a block like below:
+//
+//  Message: We must create new standard behaviour 	Function: BuildPack  |  display: red,  words: 20,
+//
+//  Message: We must create new standard behaviour 	Function: BuildPack  |  display: red,  words: 20,
```

Comment:
@adambabik This is how current log lines are displayed:

```Message: RPC Message Status Notification  Function: github.com/status-im/status-go/geth/node.(*NodeManager).startNode.func1  File: /Users/darkvoid/devlabs/src/github.com/status-im/status-go/geth/node/manager.go:99  |  STATUSIM_MESSAGE_STATE_NOTIFICATION: {"reason":null,"envelope":{"Version":"AA==","Expiry":1508427178,"TTL":20,"Topic":"0x796000e9","AESNonce":null,"Data":"BErMAvKCU4BZiAsmg226YE/WYFbnmw4C1fcvKpsUBS7jNYAJJJ3IGH/sQC+hiO3goIA0Rhq/akX7IMbBqlQWEowcV1xW6aNZmkibh6+zsyvjvqzq06s6cXolTmLEuaNwhDg3gcgn6vXQagMziPN3gb6DrfDHd7bB/2pEeoTcGBPoJHYphlFBGtS3ZitBEtCB5qJ6q+1g9GX4p/4Qen3J56pO7yd8RwGf1vA8Nxj6AQeOMp+iguyFyWczUw31Edx57Vkl8j/V0ZjhTehfOoZ7VZs7m9icsFsgrU0tbtY779boIOtnvwSVRRZn40IGb6bD3ld/MiMhJZ+/6Ct9KmsHUpgRdFLB6jPVLkjAsXq+Kz/nfnuHa7RXoN4gh9WXbaRyNlJK1h124f9tOsPyhIx/Tbe2dFfvbyPaJU1r1B1TNUl/m7nIuBWL5XDBTJ8FxOC+2F4ThVwkOq0dAKn+QxDGA6N7p+KL1Fc5zVvDksGAODKR","EnvNonce":76},"timestamp":"2017-10-19T16:32:38.461709794+01:00","Source":{"symKeyID":"","pubKey":"0x","sig":"","ttl":0,"topic":"0x00000000","payload":"0x","padding":"0x","powTime":0,"powTarget":0,"targetPeer":""},"status":8,"direction":0,"received":null}  networkID: 3  message_direction: UnknownDirection```

![image](https://user-images.githubusercontent.com/131629/31785182-416ddc02-b4fc-11e7-8d3c-ad9954ef809b.png)